app/actions/websocket/posts.test.js

```diff
@@ -62,7 +62,7 @@ describe('Websocket Post Events', () => {
         // Mock that post does not exist and check it is added
         PostSelectors.getPost.mockReturnValueOnce(false);
         mockServer.emit('message', message);
-        await TestHelper.wait(100);
+        await TestHelper.wait(300);
```

Comment:
Is there something specific to this PR makes this change necessary?

Reply:
This is a naïve way to "fix" a flaky test that was giving us problems to generate the builds. It is not inherent to this PR, but if we don't have it, we might not be able to generate the builds on CI (since tests fail).